wiki/en/Tips-Tricks-More.md

```diff
@@ -98,9 +98,9 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
 
 ## For Server admins
 
-### Converting a public Server to a private one on the fly
+### Converting a Registered Server to an Unregisterd one on the fly
```

Comment:
I think the intent here is to be _publically registered_ (i.e. on the built in Public Directories), so you're easy to find initially and then to drop off the list.  Maybe add "Publically" before "Registered"?

Reply:
This is what I mean about the "feeling" of registration being a matter of making things public or private. But again, if we use the word "public" here, there is at least some implication that "private" may also exist. I don't think we can expect people to understand such subtleties, hence my suggested wording. 

BTW if the reader doesn't understand what "Registered" and "Unregistered" mean here, they would keep reading the docs. But that might not be the case if they think they (incorrectly) they understand  that "public" means "Not password protected", for example.

Reply:
Being "registered" is what makes a server "visible" -- but only to those who have access to the directory with which it registers.  Previously all such servers were "public", but we changed that as it's clearly wrong (i.e. if you don't have access to the directory, you don't know the server exists -- unless it's publicised some other way).

However, using the built in directories _explicitly_ makes a server publically visible.  That's why we refer to the directories as public -- because the enforce that visibility upon the registering server.

Reply:
Hm, why isn't my spell check working. I've been thinking "publically" looks wrong - it's "publicly".  Tsk,